nrf_802154_sl/CHANGELOG.rst

```diff
@@ -12,11 +12,10 @@ All notable changes to this project are documented in this file.
 nRF 802.15.4 Service Layer 0.3.0
```

Comment:
It seems we should mark it here as version 0.4.0 with its changelog entry.

Reply:
Please check if the new entry is appropriate.